src/components/features/Order/PaymentInfoSection/index.tsx

```diff
@@ -62,28 +45,40 @@ export const PaymentInfoSection = ({ price, onPaymentClick }: PaymentInfoProps)
       <Divider opacity={0.6} borderWidth="0px 0px 1px" w="100%" color="rgb(237, 237, 237)" />
       <Box w="100%" padding="16px">
         <Checkbox
+          {...register('isCashReceipt')}
           cursor="pointer"
           display="inline-flex"
           alignItems="center"
           position="relative"
-          onChange={handleCheckboxChange}
         >
           현금영수증 신청
         </Checkbox>
         <Box w="100%" h="16px" />
-        <Select w="100%" pb="1px">
+        <Select w="100%" pb="1px" disabled={!isCashReceipt}>
           <option value="PERSONAL">개인소득공제</option>
           <option value="BUSINESS">사업자증빙용</option>
         </Select>
```

Comment:
```suggestion
        {isCashReceipt && <Select w="100%" pb="1px">
          <option value="PERSONAL">개인소득공제</option>
          <option value="BUSINESS">사업자증빙용</option>
        </Select>}
```

현금영수증 신청을 체크하지 않으면 보지 않아도 되는 뷰라고 생각해서 해당 select와 cashReceiptNumber를 조건부 렌더링으로 jsx에서 표현해주지 않는게 사용자의 고민을 줄여줄테니 더 좋은 선택 같아 보이긴 해요.